e2e/{{app_name}}/tests/index.spec.ts

```diff
@@ -1,10 +1,14 @@
-const { test, expect } = require('@playwright/test');
+import { Page, expect, test } from '@playwright/test';
 
 import AxeBuilder from '@axe-core/playwright';
 
 test.describe('Generic Webpage Tests', () => {
   test('should load the webpage successfully', async ({ page }) => {
     const response = await page.goto('/');
+    if (!response) {
+      throw new Error('Failed to navigate');
+    }
```

Comment:
this error doesn't seem accurate. from [docs](https://playwright.dev/docs/api/class-page#page-goto)

> The method either throws an error or returns a main resource response. The only exceptions are navigation to about:blank or navigation to the same URL with a different hash, which would succeed and return null. suggest just asserting, something like

```suggestion
    expect(response).not.toBeNull();
```

Reply:
I still see a type error if you make the above change

Reply:
or are you suggesting the entire test just be this:
```js
  test('should load the webpage successfully', async ({ page }) => {
    const response = await page.goto('/');
    const title = await page.title();
    expect(response).not.toBeNull();
  });
  ```
  - not sure we need the `const title` line - but it is kind of illustrative of how to access elements on the page

Reply:
sure adding `!` sounds good